extensions/amp-carousel/0.2/fn_test/test-carousel.js

```diff
@@ -0,0 +1,32 @@
+import 'babel-regenerator-runtime';
```

Comment:
@estherkim and I discussed folder layout and we settled on putting the end-to-end tests' JS files in the `test/e2e/` folder, and the HTML files in `amphtml/fixtures/` (maybe `amphtml/fixtures/e2e`?). This has the benefits of decoupling the test JS from the extension JS and deemphasizing e2e tests from being only "tests for this AMP extension" and emphasizing that they should be "tests for this AMP user flow". What do you think? Glad to have a quick VC sync to discuss ideas.

Reply:
I would strongly prefer that tests for a single component are beside the code they are testing. If we want to have functional tests that test multiple components together, I agree a centralized location would be the best place. Having the tests in another folder just puts them out of sight, making a developer less likely to remember to add tests as they make changes.

Reply:
Makes sense to me re: testing a single component. Can we change fn_test to e2e-test or e2e?

Reply:
Changed the folder name to `e2e-test`.

Reply:
Ah wait! Can we change it to `test-e2e` so it stays close to the other `test` folder? Sorry my bad!